presto-hive/src/test/java/io/prestosql/plugin/hive/statistics/TestMetastoreHiveStatisticsProvider.java

```diff
@@ -463,7 +463,7 @@ public void testConvertPartitionValueToDouble()
         assertConvertPartitionValueToDouble(SMALLINT, "1234", 1234);
         assertConvertPartitionValueToDouble(TINYINT, "123", 123);
         assertConvertPartitionValueToDouble(DOUBLE, "0.1", 0.1);
-        assertConvertPartitionValueToDouble(REAL, "0.2", (double) (float) 0.2);
```

Comment:
I not sure if you want to change this one.  It is processing the value through float, which might be what is expected.

Reply:
The cast to double is redundant since takes a double and thus the argument will be promoted to that automatically. But @findepi is correct this should just be `0.2f` (no need to cast a `0.2` double literal to float).